resources/client/cl_photo.ts

```diff
@@ -43,11 +43,16 @@ RegisterNuiCB<void>(PhotoEvents.TAKE_PHOTO, async (_, cb) => {
   CellCamActivate(true, true);
   // Hide phone from rendering temporary
   closePhoneTemp();
-
   SetNuiFocus(false, false);
 
   inCameraMode = true;
 
+  // We want to emit this event for UI handling in other resources
+  emit(PhotoEvents.NPWD_PHOTO_MODE_STARTED);
+
+  // Player shouldnt have radar on screen
+  DisplayRadar(false);
```

Comment:
You shoukd check if the radar is already disabled before disabling it and then assign a bool here for later use if npwd disables it.

“why rocko?”
In the event a server has no minimap, we don’t want to enable it later. This would work as the initial check to see if they had a minimap to begin with. 

You can use `IsRadarHidden()` to check. It will return false if it’s showing.